### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by jlvFlores**
In this PR, we:
- Added JS linter
- Added a functioning menu dropdown
- Added styles to the menu dropdown
- Updated README description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,13 +11,14 @@
...
-            <a href="#contact" class="nav-desktop">contact</a>
-            <img class="nav-desktop" src="assets/images/Icon - Mail.png" alt="Contact icon">
-            <img class="nav-mobile" src="assets/images/Union.png" alt="Menu icon">
+            <a class="nav nav-desktop hide" href="#portfolio">Portfolio</a>
+            <a class="nav nav-desktop hide" href="#about">About</a>
+            <a class="nav nav-desktop hide" href="#contact">contact</a>
+            <img class="nav nav-desktop hide" src="assets/images/Icon - Mail.png" alt="Contact icon">
+            <img id="menu-icon" class="nav nav-mobile" src="assets/images/Menu.png" alt="Menu icon">
+            <img id="exit-icon" class="nav nav-mobile hide" src="assets/images/Exit.png" alt="Exit icon">
         </nav>
```
Comment (TadesseDev) [inline index.html]: - [ ] Dear @jlvFlores, Your design is looking great already. But I believe there are some improvements we can make. as you can see we need the `underline` under each menu item, the `white space` on the bottom, and the message icon should not be visible on the mobile menu.


### Figma design 
Screenshot 2023-01-31 194816

### Your design
Screenshot 2023-01-31 194915

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @jlvFlores 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 
## Required Changes ♻️
- [ ] Please add a JavaScript linter  (`ESlint`) for your project. Remember This is the first project in which you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jlvFlores) [reply index.html]: Hello @Tadesse-Alemayehu thank you for your feedback, I will add the changes that you've proposed to me. However, I went over the Figma template and noticed that the white space is part of the background and not in fact an element. I could make the dropdown a bit longer to hide the headline and make it a bit better to look at, what do you think?

Edit: Also, the dropdown has an absolute position in Figma, so adding a white space that extends all the way down would also be unattractive to look at.

Comment (TadesseDev) [reply index.html]: Nice to reach out @jlvFlores, But the white part is not the background, if you notice there is a horizontal line in the middle that is going to be used to close the menu (just like the `X` close the menu). You are not expected to implement that functionality as of now, but the design should be there.

**Review by Gambit142 — COMMENTED** [review]
Hi @jlvFlores @mwismann 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- I noticed you tampered with the linters configurations for this project. Please, do not tamper with them. You should use the recommended linters configuration for each project without tampering with it. For this project kindly use the HTML & CSS3 & JavaScript linters configuration. You can find them in this [article](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -11,13 +11,17 @@
 </head>
 <body>
     <header>
-        <a href="index.html" class="logo"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
+        <a id="logo" href="index.html"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
         <nav>
```
Comment (topeogunleye) [inline index.html:15]: - [ ] [OPTIONAL] I kindly suggest that you add a position of sticky to your Navbar to make it always visible to the user even when they click on any of the mobile menu options.

**Review by topeogunleye — APPROVED** [review]
### Hi @jlvFlores, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**